moda-contracts/script/SetOfficialContracts.s.sol

```diff
@@ -15,14 +14,14 @@ contract SetOfficialContracts is Script {
 
         vm.startBroadcast(privateKey);
 
-        address registry = DeployedContracts.get("DeployRegistry.s.sol", block.chainid);
+        Registry registry = Registry(DeployedContracts.get("DeployRegistry.s.sol", block.chainid));
         IManagement management =
             IManagement(DeployedContracts.get("DeployManagement.s.sol", block.chainid));
         ISplitsFactory splitsFactory =
             ISplitsFactory(DeployedContracts.get("DeploySplitsFactory.s.sol", block.chainid));
```

Comment:
This needs to be changed to `DeployedContracts.getAt("DeploySplitsFactory.s.sol", block.chainid, 1)`